xla/service/cpu/tests/BUILD

```diff
@@ -396,12 +396,12 @@ xla_cc_test(
         "//xla:test_helpers",
         "//xla/hlo/utils:hlo_matchers",
         "//xla/service:cpu_plugin",
-        "//xla/service/cpu:onednn_contraction_rewriter",
         "//xla/service/cpu:onednn_util",
         "//xla/tests:filecheck",
         "//xla/tests:hlo_test_base",
         "//xla/tests:test_macros_header",
         "//xla/tests:xla_internal_test_main",
+        "//xla/service/cpu:onednn_contraction_rewriter",
```

Comment:
Why is this dependency moved? The list should be sorted alphabetically.

Reply:
Sorry about that! I thought the dependencies were arranged in increasing dependency string lengths, have corrected.